R/html_dependencies.R

```diff
@@ -87,9 +50,9 @@ navbar_icon_dependencies <- function(navbar) {
 html_dependencies_fonts <- function(font_awesome, ionicons) {
   deps <- list()
   if (font_awesome)
-    deps <- append(deps, list(html_dependency_font_awesome()))
+    deps <- append(deps, list(htmldeps::html_dependency_font_awesome()))
```

Comment:
You won't need any of these explicit qualifiers once you've done the re-export (since they'll all be available as functions within the rmarkdown NAMESPACE)